resorts/views/resort_list.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+#General imports
+from collections import defaultdict
+from django.core.exceptions import ObjectDoesNotExist
+
+#Import Resort and ConditionsReport models
+from ..models import Resort, Conditions
+
+#Imports for using Google Maps
+import googlemaps
+import json
+
+#Google Maps requires a client key to work
+GMAPS = googlemaps.Client(key='AIzaSyBRrCgnGFkdRY-Z1hX6xaxoUFBczNI2664')
+
+
+def get_resort_list(user_address, selected_resorts, number_to_display=5, order_on='snow_in_past_24h'):
+#Given a user address and selected resorts, return an ordered list of resorts for display
+#We first use the Google Maps API to get distances and driving times for the journey between the user and the resorts
+#We then use our scraped objects to get the 24 hr snow fall, temperatures at the mountain base, and number of trails open
+#Finally, we return the sorted information as a list of tuples
+
+    resorts_info = defaultdict(dict)
```

Comment:
Perhaps use 'resorts_list' to keep it in line with the view name (change throughout file)